src/main/java/service/imp/WriterServiceImpl.java

```diff
@@ -0,0 +1,17 @@
+package service.imp;
```

Comment:
The package name should be `service.impl` to follow the convention for implementation classes. Please correct the package name.